apps/api/src/test-data/validateTestData.spec.ts

```diff
@@ -100,6 +100,21 @@ describe('buildTestData', () => {
     });
 
     describe('test data for edge connections', () => {
+        it(`should have no duplicate IDs`, () => {
```

Comment:
Double check that the `resource` duplicate IDs check isn't within a loop here.

Reply:
But it **should** be in a loop. The IDs are only unique within a collection (whereas there is only one collection for the `EdgeConnections` so there is no loop there.